rdr_service/dao/genomics_dao.py

```diff
@@ -740,6 +740,44 @@ def get_control_sample_for_gc_and_genome_type(self, _site, genome_type, biobank_
                 GenomicSetMember.genomicWorkflowState != GenomicWorkflowState.IGNORE
             ).one_or_none()
 
+    def get_new_participants(
+        self,
+        genomic_workflow_state,
+        biobank_prefix=None,
+        genomic_set_id=None,
+        is_sql=False
+    ):
+        with self.session() as session:
+            participants = session.query(
+                sqlalchemy.bindparam('value', ''),
```

Comment:
Do they want to remove `value` as well?

Reply:
Your right, https://github.com/all-of-us/raw-data-repository/pull/2534/commits/afd7e5ab0117857f482de67a3dcbe6d2dbe69c81